bitcoin-test/lib/Bitcoin/Orphans.hs

```diff
@@ -14,15 +14,16 @@ import Bitcoin (
     OutPoint (OutPoint),
     ParsedPath (..),
     PubKeyI,
```

Comment:
Random comment:

Upstream secp256k1 actually did a bunch of renames for the various types and stuff. I liked a lot of those changes/renames. We should consider adopting some of those name changes in our fork.

Reply:
Do you have a list of the ones you like?